qirlib/CMakeLists.txt

```diff
@@ -3,6 +3,28 @@
 
 cmake_minimum_required(VERSION 3.10.0)
 
+# Copy the command line arguments to a cache variable so that they can be used
+# with the inner project. This is needed because the inner project is a
+# needs to use the same configuration and the command line arguments are not
```

Comment:
```suggestion
# with the inner project. This is needed because the inner project
# needs to use the same configuration and the command line arguments are not
```